test/test_torch.py

```diff
@@ -19182,6 +19182,12 @@ def inner(self, device, dtype):
     ('expand_as', '', _new_t((_M, 1, _M)), lambda t, d: [_new_t((_M, 4, _M))(t, d)],
         1e-5, 1e-5, 1e-5, _types, _cpu_types, False),
     ('fill_', '', _medium_2d, lambda t, d: [_number(3.14, 3, t)], 1e-3, 1e-5, 1e-5, _types, _cpu_types, False),
+    ('gcd', '', _small_3d, lambda t, d: [_small_3d(t, d)], 1e-1, 1e-0, 1e-5,
```

Comment:
For the tolerances here, can they not all be 0?

Reply:
Yes, you're right.